styles.css

```diff
@@ -1 +1,64 @@
-/*Maak hier jouw CSS!*/
```

Comment:
/*CSS Reset* deze had ik zelf ook niet maar stond wel in het voorbeeld*/
* {
    margin: 0;
    padding: 0;
}